js/profile-w3c-common.js

```diff
@@ -78,6 +78,7 @@ define(
     "w3c/permalinks",
     "core/id-headers",
     "core/rdfa",
+    "core/lint-hrefs-ids",
```

Comment:
I wonder if this should just be imported directly in the w3c/linter.js? It doesn't make sense to run it here because this check should only occur after everything else finishes (and shouldn't block rendering).